src/PackagePrivate/Humanizer/InternationalizedHumanizer.php

```diff
@@ -250,7 +250,17 @@ private function humanizeYearMonthDay( ?string $year, ?string $month, ?string $d
 		);
 	}
 
-	private function humanizeYear( int $year, UnspecifiedDigit $unspecifiedDigit ): string {
+	private function humanizeYear( int $year, ExtDate $date ): string {
+		$unspecifiedDigit = $date->getUnspecifiedDigit();
+
+		// *** if the expression $date->getDay() === null && $date->getMonth() === null
+		// is superfluous as it seems the function can be restored to its
+		// original declaration ( int $year, UnspecifiedDigit $unspecifiedDigit )
```

Comment:
We'll, find out by removing the code. We have the tests, and you can add more tests if there are cases you are not sure about.